src/ActiveCell.js

```diff
@@ -19,10 +23,10 @@ type Props<Cell, Value> = {|
   mode: Types.Mode,
   edit: () => void,
   getBindingsForCell: Types.getBindingsForCell<Cell>,
-  setCellCommit: Types.Commit => void
+  onCellCommit: Types.onCellCommit<Cell>
 |};
 
-class ActiveCell<Cell, Value> extends Component<Props<Cell, Value>> {
+class ActiveCell<Cell, Value> extends Component<Props<Cell, Value>, State<*>> {
```

Comment:
Shouldn't `State` get `Cell`?

Reply:
It should, but flow shouts at me, so I allowed him to infer

Reply:
ok